### PR TITLE
Chores/listingdetails endpoint #113977289

### PR DESCRIPTION
**Pull request by kaptron**
PR app won't work unless we change credentials to sandbox since `ListingDetails` endpoint is not on salesforce prod

### CODE REVIEW

**Comment by kaptron** [comment]
**DO NOT MERGE** until salesforce `ListingsDetail` endpoint has been pushed to prod

File: app/services/salesforce_service.rb [hunk 1]
```diff
@@ -44,7 +44,12 @@ def self.eligible_listings(filters)
 
   # get one detailed listing result by id
   def self.listing(id)
-    api_get("/services/apexrest/Listings/#{id}").first
+    api_get("/services/apexrest/ListingDetails/#{id}").first
+  end
+
+  # get all units for a given listing
+  def self.units(listing_id)
+    api_get("/services/apexrest/Listing/Units/#{listing_id}")
```
Comment (chrisdolendo) [inline app/services/salesforce_service.rb]: not really for you but why isn't vertiba being RESTful? `/listings/listing_id/units`... haha

**Comment by chrisdolendo** [comment]
coolio :+1:

**Comment by kaptron** [comment]
This PR now contains all of the work done with the various (now accepted) feature branches:
- #137 
- #139 
- #142

**Pull request merged**